dev/tests/integration/testsuite/Magento/Bundle/_files/order_with_bundle_shipped_together.php

```diff
@@ -0,0 +1,212 @@
+                'type' => 'select',
+                'required' => 1,
+                'delete' => '',
+            ],
+        ]
+    )
+    ->setBundleSelectionsData(
+        [
+            [
+                [
+                    'product_id' => $sampleProduct->getId(),
+                    'selection_qty' => 1,
+                    'selection_can_change_qty' => 1,
+                    'delete' => '',
+                ],
+            ],
+            [
+                [
+                    'product_id' => $secondSampleProduct->getId(),
+                    'selection_qty' => 1,
+                    'selection_can_change_qty' => 1,
+                    'delete' => '',
+                ],
+            ]
+        ]
+    );
+
+if ($product->getBundleOptionsData()) {
+    $options = [];
+    foreach ($product->getBundleOptionsData() as $key => $optionData) {
+        if (!(bool)$optionData['delete']) {
+            $option = $objectManager->create(\Magento\Bundle\Api\Data\OptionInterfaceFactory::class)
+                ->create(['data' => $optionData]);
+            $option->setSku($product->getSku());
+            $option->setOptionId(null);
+
+            $links = [];
+            $bundleLinks = $product->getBundleSelectionsData();
+            if (!empty($bundleLinks[$key])) {
+                foreach ($bundleLinks[$key] as $linkData) {
+                    if (!(bool)$linkData['delete']) {
+                        /** @var \Magento\Bundle\Api\Data\LinkInterface $link */
+                        $link = $objectManager->create(\Magento\Bundle\Api\Data\LinkInterfaceFactory::class)
+                            ->create(['data' => $linkData]);
+                        $linkProduct = $productRepository->getById($linkData['product_id']);
+                        $link->setSku($linkProduct->getSku());
+                        $link->setQty($linkData['selection_qty']);
+                        if (isset($linkData['selection_can_change_qty'])) {
+                            $link->setCanChangeQuantity($linkData['selection_can_change_qty']);
+                        }
+                        $links[] = $link;
+                    }
+                }
+                $option->setProductLinks($links);
+                $options[] = $option;
+            }
+        }
+    }
+    $extension = $product->getExtensionAttributes();
+    $extension->setBundleProductOptions($options);
+    $product->setExtensionAttributes($extension);
+}
+$product->save();
+
+$objectManager = \Magento\TestFramework\Helper\Bootstrap::getObjectManager();
+
+$addressData = include __DIR__ . '/../../../Magento/Sales/_files/address_data.php';
+
+$billingAddress = $objectManager->create(\Magento\Sales\Model\Order\Address::class, ['data' => $addressData]);
+$billingAddress->setAddressType('billing');
+
+$shippingAddress = clone $billingAddress;
+$shippingAddress->setId(null)->setAddressType('shipping');
+
+$payment = $objectManager->create(\Magento\Sales\Model\Order\Payment::class);
```

Comment:
Please use $objectManager->get(...) in the whole file where in is posible.

Reply:
done